python/cudf/cudf/core/dataframe.py

```diff
@@ -3598,6 +3602,20 @@ def kurtosis(self, axis=None, skipna=None, level=None, numeric_only=None):
             skipna=skipna,
             level=level,
             numeric_only=numeric_only,
+            **kwargs,
+        )
+
+    def kurt(
```

Comment:
Should be enough just to do `kurt = kurtosis` at the class level.

Reply:
Done 👍